### PR TITLE
Update README for MNIST example

### PR DESCRIPTION
**Pull request by VMS-6511**

### CODE REVIEW

File: examples/mnist/README.md [hunk 1]
```diff
@@ -34,32 +34,55 @@ class ModelOwner:
...
-
-w0, b0, w1, b1 = tfe.define_private_input(model_owner.player_name, model_owner.provide_input)
-x = tfe.define_private_input(prediction_client.player_name, prediction_client.provide_input)
-
-layer0 = tfe.matmul(x, w0) + b0
-layer1 = tfe.sigmoid(layer0)
-logits = tfe.matmul(layer1, w1) + b1
-
-prediction_op = tfe.define_output(prediction_client.player_name, logits, prediction_client.receive_output)
+  model_owner = ModelOwner(
```
Comment (yanndupis) [inline examples/mnist/README.md]: I'd keep as well the following lines to reduce the level of details. 
```
model_owner = ModelOwner('model-owner')
prediction_client = PredictionClient('prediction-client')
```

File: examples/mnist/README.md [hunk 2]
```diff
@@ -34,32 +34,55 @@ class ModelOwner:
...
+      local_data_file="./data/train.tfrecord")
+
+  prediction_client = PredictionClient(
+      player_name="prediction-client",
+      local_data_file="./data/test.tfrecord")
+
+  with tfe.protocol.SecureNN():
+    batch_size = PredictionClient.BATCH_SIZE
+    flat_dim = ModelOwner.IMG_ROWS * ModelOwner.IMG_COLS
+    batch_input_shape = [batch_size, flat_dim]
```
Comment (yanndupis) [inline examples/mnist/README.md]: You can remove how we define `batch_input_shape` to simplify.

File: examples/mnist/README.md [hunk 3]
```diff
@@ -34,32 +34,55 @@ class ModelOwner:
...
+
+    model = tfe.keras.Sequential()
```
Comment (yanndupis) [inline examples/mnist/README.md]: we don't show that the model owner provides the model weights 
```
params = tfe.define_private_input(model_owner.player_name,
                                    model_owner.provide_input)
```

we should also add `model.set_weights(params, sess)` after the last model.add

File: examples/mnist/README.md [hunk 4]
```diff
@@ -34,32 +34,55 @@ class ModelOwner:
...
 
     def provide_input(self) -> tf.Tensor:
-        prediction_input = self._build_data_pipeline().get_next()
-        # pre-processing
-        prediction_input = tf.reshape(prediction_input, shape=(self.BATCH_SIZE, 28 * 28))
+        """Prepare input data for prediction."""
+        with tf.name_scope('loading'):
+        prediction_input, expected_result = self._build_data_pipeline().get_next()
+        print_op = tf.print("Expect", expected_result, summarize=self.BATCH_SIZE)
+        with tf.control_dependencies([print_op]):
```
Comment (yanndupis) [inline examples/mnist/README.md]: for this section, I would remove `tf.name_scope`, print_op, `tf.control_dependencies`. It's too many details for the  README. I would suggest to match the same level of details as before.

File: examples/mnist/README.md [hunk 5]
```diff
@@ -34,32 +34,55 @@ class ModelOwner:
...
+
+  with tfe.protocol.SecureNN():
+    batch_size = PredictionClient.BATCH_SIZE
+    flat_dim = ModelOwner.IMG_ROWS * ModelOwner.IMG_COLS
+    batch_input_shape = [batch_size, flat_dim]
+
+    model = tfe.keras.Sequential()
+    model.add(tfe.keras.layers.Dense(512, batch_input_shape=batch_input_shape))
+    model.add(tfe.keras.layers.Activation('relu'))
+    model.add(tfe.keras.layers.Dense(10, activation=None))
```
Comment (jvmancuso) [inline examples/mnist/README.md]: I believe `activation=None` is unnecessary

File: examples/mnist/README.md [hunk 6]
```diff
@@ -34,32 +34,43 @@ class ModelOwner:
...
+
+  with tfe.protocol.SecureNN():
+    batch_size = PredictionClient.BATCH_SIZE
+    flat_dim = ModelOwner.IMG_ROWS * ModelOwner.IMG_COLS
+
+    model = tfe.keras.Sequential()
+    model.add(tfe.keras.layers.Dense(512, batch_input_shape=batch_input_shape))
+    model.add(tfe.keras.layers.Activation('relu'))
+    model.add(tfe.keras.layers.Dense(10))
+    model.set_weights(params, sess)
```
Comment (yanndupis) [inline examples/mnist/README.md]: Here you can remove `sess` from `set_weights` for now. thanks

**Comment by yanndupis** [comment]
@VMS-6511 LGTM! thank you. just please squash and merge.

**Pull request merged**